automerge-backend/src/backend.rs

```diff
@@ -135,9 +135,25 @@ impl Backend {
         }
 
         let bin_change: Change = change.into();
-        let patch: amp::Patch = self.apply(vec![bin_change.clone()], Some(actor_seq))?;
+        let hash = bin_change.hash;
 
-        Ok((patch, bin_change))
+        let patch: amp::Patch = self.apply(vec![bin_change], Some(actor_seq))?;
+
+        let change = self.get_change_by_hash(&hash).unwrap();
+
+        Ok((patch, change))
+    }
+
+    pub fn apply_local_change_mut(
```

Comment:
Can we document that you would typically use this if you want to compress the change?

Reply:
Yep, done.